flutter_local_notifications_windows/lib/src/details/notification_part.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:xml/xml.dart';
+
+/// A text or image element in a Windows notification.
+///
+/// Note: This should not be used for anything else as notification
+/// groups can only contain text and images.
+// This class needs to be abstract so [WindowsNotificationText] and
+// [WindowsImage] can extend it. Specifically, this class is a marker
+// type for classes that are valid as part of a [WindowsColumn].
+// ignore: one_member_abstracts
+abstract class WindowsNotificationPart {
+  /// A const constructor.
+  const WindowsNotificationPart();
+
+  /// Serializes this part to XML, according to the Windows API.
+  void toXml(XmlBuilder builder);
```

Comment:
Can you move this out so it's not part of the public API surface? Could you also see if this could be renamed? It doesn't return a result and methods named `to<...>` conventionally return a result due to how they're named